src/picterra/__main__.py

```diff
@@ -207,6 +207,15 @@ def create_remote_raster_type(coords_string):
     delete_detectionarea_parser.add_argument(
         "raster", help="ID of the raster whose detection areas will be deleted", type=str)
 
+    # create the parsers for the "edit" command
+    edit_parser = subparsers.add_parser('edit', help="Edit resources")
+    edit_subparsers = edit_parser.add_subparsers(dest='edit')
+    # edit raster
+    edit_raster_parser = edit_subparsers.add_parser('raster', help="Edits a raster")
+    edit_raster_parser.add_argument("raster", help="ID of the raster to edit", type=str)
+    edit_raster_parser.add_argument(
+        "--name", help="Name to give to the raster", type=str, required=False)
+
```

Comment:
Might not be worth the change, but I believe you could also separate all of the business logic code in e.g. a separate file, and write the parser as:
```
<subparser_creation>
<add_arguments>
edit_raster_parser.set_defaults(func=mymodule.myfunction)
```

And later in `__main__.py`:
```
args = parser.parse_args()
args.func(args)
```

That would ease up a bit the other half of this file (the various `if options.xxx`).

This pattern is documented in [argparse](https://docs.python.org/3/library/argparse.html) (search for "func="). I've used it in our [skywatch-connector](https://gitlab.com/picterra/skywatch-connector/-/blob/master/skywatch.py),

Reply:
nice! didn't know about it, will do like this, thanks!

Reply:
thanks a lot, looks nicer now!

Reply:
Looks great! Any reason why you didn't follow the `set_defaults()` pattern above and relied instead on `CommandParser`?

I've just noted 2 things:
- `command_helpers` is missing a `__init__.py` file. The cli doesn't run without this (shouldn't this be caught by tests somehow?)
- You could maybe rename `command_helpers` to `command_parsers`, since that's what's in there.

Reply:
done